src/kudu/consensus/leader_election.cc

```diff
@@ -260,7 +260,7 @@ FlexibleVoteCounter::FlexibleVoteCounter(
   // Computes voter distribution and uuid to region map.
   FetchTopologyInfo();
 
-  for (const std::pair<std::string, int>& regional_voter_count :
+  for (const std::pair<const std::string, int>& regional_voter_count :
```

Comment:
When iterating through a list, key is defined as `const` to avoid modifications.

Current code forces a non-const key which actually results in a copy despite the reference modifier, which is a compiler warning error in fbcode.